### ISSUE TITLE
Stylelint files missing from instructions in docs/porting-addons-to-v2.md

### ISSUE THREAD

**Issue by fdeters** [embroider-build/embroider#1562]
Hello! I noticed that `.stylelintrc.js` and `.stylelintignore` are missing from the instructions in step 18 of part 1 of the v2 addon porting instructions:

> 18. At this point all tests and lint scripts work the same way as before inside the test-app. But we will also want linting, prettier, etc for the newly-separated addon workspace too.
> _..._ 
> Copy **.gitignore, .eslintrc.js, .eslintignore, .prettierrc.js, .prettierignore, and .template-lintrc.js** from test-app to addon. 

Without copying those two files over to the addon, the lint:css scripts don't work.

**Comment by fdeters** [comment]
Looking at it again, the real problem is probably that the stylelint packages are missing from this section of step 18 as well:

> Copy **eslint, relevant eslint plugins, prettier, ember-template-lint, and npm-run-all** from `test-app/package.json` `devDependencies` to `addon/package.json` `devDependencies`.

I believe stylelint, stylelint-config-standard, and stylelint-prettier all need to be moved?

**Comment by NullVoxPopuli** [comment]
yes -- stylelint was added to the blueprint after that document was written 🙃
